### PR TITLE
Run3-gex110 Recover BHM placements in the dd4hep scenario

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Recover BHM placements in the dd4hep scenario

#### PR validation:

Use test workflows in Geometry/ForwardCommonData/test/python

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by civanch** [comment]
+1

**Comment by perrotta** [comment]
+1

File: Geometry/ForwardCommonData/data/pixfwd.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0"?>
```
Comment (cvuosalo) [inline Geometry/ForwardCommonData/data/pixfwd.xml:1]: This file should have a version number in the directory name, maybe `Geometry/ForwardCommonData/data/forward/2021/v1/pixfwd.xml`?

**Comment by tvami** [comment]
Hi @bsunanda does this need to be backported to 12_2_X? (Sorry if it was, I didnt see the link to this PR)

**Comment by bsunanda** [comment]
This PR is backported and is apart of #36740 and is now documented in its history
________________________________

**Comment by perrotta** [comment]
(Repeating also here for completeness)
I'm sorry @bsunanda , but none of the files modified here or in #36663 is included in the backport PR #36740
Could you please check, and provide the necessary amendments if needed?

**Pull request merged**